tests/ignite/contrib/handlers/test_clearml_logger.py

```diff
@@ -3,13 +3,14 @@
 from collections import defaultdict
 from unittest.mock import ANY, MagicMock, Mock, call
 
+import clearml
 import pytest
 import torch
-import trains
-from trains.binding.frameworks import WeightsFileHandler
-from trains.model import Framework
+from clearml.binding.frameworks import WeightsFileHandler
+from clearml.model import Framework
 
 import ignite.distributed as idist
+from ignite.contrib.handlers.clearml_logger import *
 from ignite.contrib.handlers.trains_logger import *
```

Comment:
how shall we test aliases, `TrainsLogger` and `TrainsSaver`?

Reply:
Just creation of an instance and checking that it is a new class:
```python
    with pytest.warns(UserWarning, match="ClearMLSaver: running in bypass mode"):
        TrainsLogger.set_bypass_mode(True)
        logger = TrainsLogger(output_uri=dirname)
        assert isinstance(logger, ClearMLLogger)
```